lib/streamlit/type_util.py

```diff
@@ -30,7 +30,9 @@
     cast,
     overload,
 )
+from typing import Callable
 
+from typing_extensions import ClassVar
```

Comment:
nit: doesn't look like these two imports are used

Reply:
Thanks for noticing! Was obviously moving a bit too fast here.